StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Views/PayWithLinkButton.swift

```diff
@@ -467,7 +469,9 @@ private func makeAccountStub(email: String, isRegistered: Bool, lastPM: LinkPMDi
     return PayWithLinkButton.LinkAccountStub(
         email: email,
         redactedPhoneNumber: nil,
-        isRegistered: isRegistered
+        isRegistered: isRegistered,
+        sessionState: .verified,
+        consumerSessionClientSecret: nil
```

Comment:
The changes in this file are simply to satisfy the new protocol requirements. This method creates `LinkAccountStub`s that are used in the SwiftUI previews below, for context, and the newly exposed properties are not used.